Lib/shutil.py

```diff
@@ -489,10 +490,15 @@ def _copytree(entries, src, dst, symlinks, ignore, copy_function,
     return dst
 
 def copytree(src, dst, symlinks=False, ignore=None, copy_function=copy2,
-             ignore_dangling_symlinks=False):
+             ignore_dangling_symlinks=False, dirs_exist_ok=False):
     """Recursively copy a directory tree.
 
-    The destination directory must not already exist.
+    Copy an entire directory tree rooted at src to a directory named dst
```

Comment:
This is a repetition of the above statement "(Recursively) copy a directory tree.". You can join the 2 together and avoid repetition.

Reply:
I get your point. Then I would suggest removing the repetition in the next paragraph and just describe the new argument. To be clear, what I'm suggesting is this:

```
"""Recursively copy a directory tree.

dirs_exist_ok dictates whether to raise an exception in case dst or any
missing parent directory already exists.
...
"""
```

Reply:
Done in the latest revision.